python/ray/data/_internal/execution/interfaces/ref_bundle.py

```diff
@@ -125,3 +125,12 @@ def __hash__(self) -> int:
 
     def __len__(self) -> int:
         return len(self.blocks)
+
+
+def _ref_bundles_iterator_to_block_refs_list(
+    ref_bundles: Iterator[RefBundle],
+) -> List[ObjectRef[Block]]:
+    """Convert an iterator of RefBundles to a list of object references to Blocks."""
```

Comment:
Nit: Change to "Convert an iterator of RefBundles to a list of Block object references." to avoid the double to, for a sec thought this was a double transformation.